src/server/network/server.h

```diff
@@ -37,6 +40,7 @@ class Server final : public NetworkHost {
     glm::vec3 findPlayerSpawnPosition();
 
     void sendChunk(peer_id_t peerId, const ChunkPosition &chunk);
+    void sendPlayerSkin(peer_id_t peerId, peer_id_t toPeer = NULL);
```

Comment:
```suggestion
    void sendPlayerSkin(peer_id_t peerId, peer_id_t toPeer = 0);
```

Reply:
Isn't 0 a valid peer_id (i.e. the first peer to join gets assigned 0)?

Reply:
I want it to default it with an invalid peer_id and if so, then it broadcasts to everyone